middlewares/validators/user.js

```diff
@@ -173,8 +173,8 @@ async function validateUserQueryParams(req, res, next) {
       state: joi
         .alternatives()
         .try(
-          joi.string().valid("IDLE", "OOO", "ACTIVE"),
-          joi.array().items(joi.string().valid("IDLE", "OOO", "ACTIVE"))
+          joi.string().valid("IDLE", "OOO", "ACTIVE", "ONBOARDING"),
+          joi.array().items(joi.string().valid("IDLE", "OOO", "ACTIVE", "ONBOARDING"))
```

Comment:
since both are using the same list. we can make a constant and then use that list constant.